public/js/components/FormFields/FormFieldCheckbox.js

```diff
@@ -5,38 +5,25 @@ import { errorPropType } from '../../constants/errorPropType';
 export default class FormFieldCheckbox extends React.Component {
 
   static propTypes = {
-
-    fieldLabel: PropTypes.string.isRequired,
+    fieldLabel: PropTypes.string,
     fieldName: PropTypes.string.isRequired,
-    fieldValue: PropTypes.array.isRequired,
+    fieldValue: PropTypes.bool.isRequired,
     fieldErrors: PropTypes.arrayOf(errorPropType),
     onUpdateField: PropTypes.func.isRequired
-  };
-
-  onUpdate = (e) => {
-    this.props.onUpdateField(e.target.value);
-  }
-
-  renderCheckbox(value, i) {
-    return (
-      <div key={i} className="form__group form__group--checkbox">
-        <input className="form__checkbox" type="checkbox" checked={value.selected} name={this.props.fieldName} value={value} onChange={this.onUpdate} />
-        <span className="form__label form__label--checkbox">{value.id}</span>
-      </div>
-    );
   }
 
-  renderCheckboxes() {
-    return this.props.fieldValue.map(this.renderCheckbox, this);
+  onUpdate = () => {
+    this.props.onUpdateField(this.props.fieldName);
```

Comment:
@clloyd True, but I was thinking of the data needed for the group, would that not be a list of the names of checkboxes which are true?